code/game/machinery/cryopod_vr.dm

```diff
@@ -1,5 +1,13 @@
 //Overrides!
 
+/obj/machinery/cryopod
+	// The corresponding spawn point type that user despawning here will return at next round.
+	// Note: We use a type instead of name so that its validity is checked at compile time.
+	var/spawnpoint_type = /datum/spawnpoint/cryo
+
+/obj/machinery/cryopod/robot
+	spawnpoint_type = /datum/spawnpoint/cyborg
+
```

Comment:
"Your chosen spawnpoint (Cyborg Storage) is unavailable for your chosen job. Spawning you at the Arrivals shuttle instead."